digid_eherkenning/models/base.py

```diff
@@ -29,17 +32,29 @@ class BaseConfiguration(SingletonModel):
     )
     idp_metadata_file = PrivateMediaFileField(
         _("identity provider metadata"),
-        blank=False,
-        help_text=_("The metadata file of the identity provider."),
+        blank=True,
+        help_text=_(
+            "The metadata file of the identity provider. This is auto populated "
+            "by the retrieved metadata XML file."
```

Comment:
```suggestion
            "The metadata file of the identity provider. This is auto populated "
            "from the configured source URL."
```
& please update other help texts where a similar text is present.